tests/integration/renderers/test_pydsl.py

```diff
@@ -23,26 +23,40 @@ def test_rendering_includes(self):
         inability to load custom modules inside the pydsl renderers. This
         is a FIXME.
         '''
-        try:
```

Comment:
It was there to delete the output file... but if the test failed, the finally block wasn't being evaluated for some reason. I don't think it was really there to catch an error...